marketplace/src/app/app-routing.module.ts

```diff
@@ -1,8 +1,16 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
+import { UserActionsComponent } from './components/user-actions/user-actions.component';
+import { SellItemComponent } from './components/sell-item/sell-item.component';
+import { BuyItemComponent } from './components/buy-item/buy-item.component';
 
-const routes: Routes = [];
+
+const routes: Routes = [
+  { path: '', component: UserActionsComponent },
```

Comment:
What would the path of the landing page be?

Reply:
I imagine we would change landing to '' later, but rn just making it the useractions since not sure if we are going to include that landing page.